functions/lib/move_inactive_events.py

```diff
@@ -70,7 +70,7 @@ def get_and_move_private_inactive_events(today: date):
 
 
 
-@scheduler_fn.on_schedule(schedule="every day 00:00", region="australia-southeast1")
+@scheduler_fn.on_schedule(schedule="every day 00:00", region="australia-southeast1", time_zone="Australia/Sydney")
```

Comment:
nice : D, we just need to make sure w deploy it - it might be worthwhile to split this into another PR so we don't lose track of this change